apps/full-stack-tests/src/components/favorites/FavoriteProperties.test.ts

```diff
@@ -52,6 +52,8 @@ describe("Favorite properties", () => {
 
   describe("favoriting different types of properties", () => {
     beforeEach(async () => {
+      await Presentation.favoriteProperties.initializeConnection(imodel);
```

Comment:
how come linter doesn't complain about usage of deprecated API?

Reply:
Eventually it caught up with me and started complaining. 😆 Added ignore.